src/states_screens/track_info_screen.cpp

```diff
@@ -419,28 +447,35 @@ void TrackInfoScreen::onEnterPressedInternal()
     if (has_AI)
        num_ai = m_ai_kart_spinner->getValue();
 
-
-    if (UserConfigParams::m_num_karts_per_gamemode
-        [race_manager->getMinorMode()] != unsigned(local_players + num_ai))
-    {
-        race_manager->setNumKarts(local_players + num_ai);
-        UserConfigParams::m_num_karts_per_gamemode[race_manager->getMinorMode()] = local_players + num_ai;
-    }
-
     const int selected_target_type = m_target_type_spinner->getValue();
     const int selected_target_value = m_target_value_spinner->getValue();
 
-    if (race_manager->isSoccerMode())
+    const bool enable_ffa = m_show_ffa_spinner && selected_target_type != 0;
+
+	if (enable_ffa)
```

Comment:
Be careful, you are using tabs around here to indent. Please indent everything with spaces

Reply:
Oh sorry, I don't know where a tab came from I set my IDE to use spaces. Will check everything again for this